src/engraving/libmscore/dynamic.cpp

```diff
@@ -692,4 +692,29 @@ String Dynamic::screenReaderInfo() const
     }
     return String(u"%1: %2").arg(EngravingItem::accessibleInfo(), s);
 }
+
+DynamicType Dynamic::reconstructTypeFromString() const
```

Comment:
Yeah I originally put it in dynamic.cpp because `dynList[]` was declared as a C-style array and passing it outside the class is a pain. Anyway, I've now refactored it an std::vector static member, so I can easily use it outside 👍